generator/bindata.go

```diff
@@ -18,7 +18,7 @@
 // templates/schematype.gotmpl (854B)
 // templates/schemavalidator.gotmpl (31.005kB)
 // templates/server/builder.gotmpl (17.451kB)
-// templates/server/configureapi.gotmpl (6.265kB)
+// templates/server/configureapi.gotmpl (6.421kB)
```

Comment:
@h0mjam hey, thanks for this update.
I see a lot of differences with the generated bindata.
We are using kevin's burke fork github.com/kevinburke/go-bindata
Can you confirm you used that one? Thanks.

https://github.com/go-swagger/go-swagger/blob/master/docs/guidelines/README.md#update-templates

Reply:
I'am install `go-bindata` from brew. go-bindata should be github.com/kevinburke/go-bindata. Can do re-generate if needed.